cluster-autoscaler/core/static_autoscaler.go

```diff
@@ -402,7 +411,7 @@ func (a *StaticAutoscaler) onEmptyCluster(status string, emitEvent bool) errors.
 	if emitEvent {
 		a.AutoscalingContext.LogRecorder.Eventf(apiv1.EventTypeWarning, "ClusterUnhealthy", status)
 	}
-	return nil
+	return true
```

Comment:
Yes, it's also a way of writing code in a slightly shorter way, which I kept from a previous PR.
It also has an advantage that if you change the return type/value, you can do it once (here) and not in all the places where it's called (only two in this case, I admit).
But the main reasoning is to hide the complexity (even if it's tiny) from the main function and have a more "intent-descriptive" code there.

Please let me know if you're OK with this or would you prefer to move the return to where the method is called.

Reply:
TL;DR please change it if you don't mind.

Longer reply (only if you have the time/care to discuss this):
> have a more "intent-descriptive" code there
What intent does returning true here describe? I'm at a loss, as I don't see it representing anything. 
>It also has an advantage that if you change the return type/value, you can do it once (here) and not in all the places where it's called
If you change the return value, i.e if this ever returns false, does it mean the cluster isn't empty after all, even though we check the condition before calling it? Sounds fishy (and potentially hacky)
As for changing the return type, you can only change it once (here) if and only if all of the places where it's called are always changing the same way at the same time. So instead of callers depending on this method, this method depends on a specific set of callers. Admittedly, we happen to know only one other method calls it in this case. But reading this code still requires figuring out what value it returns and why (and realizing it's always constant.)

Disclaimer: I realize this is a tiny, tiny function that doesn't really matter this much. I still think we shouldn't introduce it like this, especially as this change is about improving the codebase :)

Reply:
Changed this.
As for caring about discussions - I'm still new here so I always like more discussion to get a better understanding of project customs.